### PR TITLE
create build-wasm

### PR DESCRIPTION
**Pull request by Toine-ddt**

### CODE REVIEW

**Comment by jcelerier** [comment]
it seems that the build fails, let's wait until you can make it work before merging ! you can see it in the "checks" panel in the PR

**Comment by Toine-ddt** [comment]
ok
I will work on it !

File: .github/workflows/build-wasm.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    container:
+      image: archlinux:latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build Docker
+        run: |
+          clone https://github.com/guerinoni/qt-wasm.git
```
Comment (jcelerier) [inline .github/workflows/build-wasm.yml:18]: git clone

File: .github/workflows/build-wasm.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: Build wasm app
+
+on: push
+
+jobs:
+  buster:
+    name: wasm build
+    runs-on: ubuntu-latest
+    container:
+      image: archlinux:latest
```
Comment (jcelerier) [inline .github/workflows/build-wasm.yml:10]: you should be able to just replace archlinux:latest by qt-wasm:latest here actually

**Comment by Toine-ddt** [comment]
@jcelerier I don't know how to solve this error :
https://github.com/ossia/qml-remote/runs/2885573326?check_suite_focus=true#step:4:28

File: .github/workflows/build-wasm.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install sudo package
+        run: apt update && apt install sudo
+
+      - name: Add docker
+        run: sudo groupadd docker
+
+      - name: Build Docker
```
Comment (jcelerier) [inline .github/workflows/build-wasm.yml:22]: since we already run in the docker image we don't need this step

Comment (jcelerier) [reply .github/workflows/build-wasm.yml:22]: (or the previous one)

File: .github/workflows/build-wasm.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        run: |
+          git clone https://github.com/guerinoni/qt-wasm.git
+          cd qt-wasm
+          chmod -R 777 /__w/_temp/
+          sudo docker build -t qtwa .
+
+      - name: Create Container
+        run: |
+          docker run -it -v /home/user/qml-remote:/remote --name webassembly qtwa
```
Comment (jcelerier) [inline .github/workflows/build-wasm.yml:31]: nor this one

Comment (Toine-ddt) [reply .github/workflows/build-wasm.yml]: ok

**Comment by jcelerier** [comment]
all good, thanks :)

**Pull request merged**